types/interfaces.ts

```diff
@@ -145,3 +141,23 @@ export interface RenderInterface {
   splitSchema: SplitSchema
   setSplitSchema: Dispatch<SetStateAction<SplitSchema>>
 }
+
+export type DocHeading = {
+  id: string
```

Comment:
This `id` seems to only be used as a React key, why not instead use the `slug` and remove the `id` entirely?

Reply:
Done